submissions/develabr/frogger/app.js

```diff
@@ -0,0 +1,116 @@
+let score = 0;
+
+class Enemy {
+  constructor(x, y, speed) {
+    this.sprite = 'images/enemy-bug.png';
+    this.x = x;
+    this.y = y;
+    this.width = 80;
+    this.height = 80;
+    this.speed = speed;
+  }
+  update(dt) {
+    this.x += this.speed * dt;
+
+    if (this.x > 510) {
+      this.x = -70;
+    }
+
+    if (
+      player.x + this.width > this.x &&
+      player.x < this.x + this.width &&
+      player.y + this.height > this.y &&
+      player.y < this.y + this.height
+    ) {
+      player.toStart();
+    }
+  }
```

Comment:
Requirements posted/linked by bot address two issues with method `update` specifically.
Please identify those, copy/paste them as a comment in this PR and fix code accrodingly.